tests/test_base_config.py

```diff
@@ -739,3 +739,18 @@ def test_assign(parent: Any, key: Union[str, int], value: Any, expected: Any) ->
 )
 def test_get_node(cfg: Any, key: Any, expected: Any) -> None:
     assert cfg._get_node(key) == expected
+
+
+@pytest.mark.parametrize(
+    "cfg, key",
+    [
+        # dict
+        (OmegaConf.create({"foo": "???"}), "foo"),
+        # list
+        (OmegaConf.create([10, "???", 30]), 1),
+        (OmegaConf.create([10, "???", 30]), slice(1, 2)),
+    ],
+)
+def test_get_node_throw_on_missing(cfg: Any, key: Any) -> None:
+    with pytest.raises(MissingMandatoryValue):
```

Comment:
```suggestion
    with pytest.raises(
        MissingMandatoryValue,
        match="Missing (mandatory value|node) " + re.escape(f"'{key}'"),
    ):
```

Reply:
Meh, I just normalized the exception message :)
Why use a toopick when you can use a BFG7000?